api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java

```diff
@@ -14,33 +14,33 @@
 @AllArgsConstructor
```

Comment:
Looks like only spacing changes on this file. No need to commit.

Reply:
Yes, it is not required.  Do you want another commit into this PR to revert it?

Reply:
Not necessary.